lua/astrocommunity/utility/nvim-toggler/init.lua

```diff
@@ -0,0 +1 @@
+return { "nguyenvukhang/nvim-toggler", event = "User Astrofile" }
```

Comment:
Do you need to run `setup`? Passing in `opts = {}`

Reply:
Fixed